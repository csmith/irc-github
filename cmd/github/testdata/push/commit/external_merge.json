{
  "ref": "refs/heads/dependabot/gradle/org.jetbrains.kotlin-kotlin-gradle-plugin-1.3.71",
  "before": "4a22fe3bf1fff68ccceee6cbf25fdc8834e0d480",
  "after": "0000000000000000000000000000000000000000",
  "repository": {
    "id": 134196659,
    "node_id": "MDEwOlJlcG9zaXRvcnkxMzQxOTY2NTk=",
    "name": "ig",
    "full_name": "greboid/ig",
    "private": false,
    "owner": {
      "name": "greboid",
      "email": "greboid@users.noreply.github.com",
      "login": "greboid",
      "id": 189618,
      "node_id": "MDQ6VXNlcjE4OTYxOA==",
      "avatar_url": "https://avatars0.githubusercontent.com/u/189618?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/greboid",
      "html_url": "https://github.com/greboid",
      "followers_url": "https://api.github.com/users/greboid/followers",
      "following_url": "https://api.github.com/users/greboid/following{/other_user}",
      "gists_url": "https://api.github.com/users/greboid/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/greboid/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/greboid/subscriptions",
      "organizations_url": "https://api.github.com/users/greboid/orgs",
      "repos_url": "https://api.github.com/users/greboid/repos",
      "events_url": "https://api.github.com/users/greboid/events{/privacy}",
      "received_events_url": "https://api.github.com/users/greboid/received_events",
      "type": "User",
      "site_admin": false
    },
    "html_url": "https://github.com/greboid/ig",
    "description": null,
    "fork": false,
    "url": "https://github.com/greboid/ig",
    "forks_url": "https://api.github.com/repos/greboid/ig/forks",
    "keys_url": "https://api.github.com/repos/greboid/ig/keys{/key_id}",
    "collaborators_url": "https://api.github.com/repos/greboid/ig/collaborators{/collaborator}",
    "teams_url": "https://api.github.com/repos/greboid/ig/teams",
    "hooks_url": "https://api.github.com/repos/greboid/ig/hooks",
    "issue_events_url": "https://api.github.com/repos/greboid/ig/issues/events{/number}",
    "events_url": "https://api.github.com/repos/greboid/ig/events",
    "assignees_url": "https://api.github.com/repos/greboid/ig/assignees{/user}",
    "branches_url": "https://api.github.com/repos/greboid/ig/branches{/branch}",
    "tags_url": "https://api.github.com/repos/greboid/ig/tags",
    "blobs_url": "https://api.github.com/repos/greboid/ig/git/blobs{/sha}",
    "git_tags_url": "https://api.github.com/repos/greboid/ig/git/tags{/sha}",
    "git_refs_url": "https://api.github.com/repos/greboid/ig/git/refs{/sha}",
    "trees_url": "https://api.github.com/repos/greboid/ig/git/trees{/sha}",
    "statuses_url": "https://api.github.com/repos/greboid/ig/statuses/{sha}",
    "languages_url": "https://api.github.com/repos/greboid/ig/languages",
    "stargazers_url": "https://api.github.com/repos/greboid/ig/stargazers",
    "contributors_url": "https://api.github.com/repos/greboid/ig/contributors",
    "subscribers_url": "https://api.github.com/repos/greboid/ig/subscribers",
    "subscription_url": "https://api.github.com/repos/greboid/ig/subscription",
    "commits_url": "https://api.github.com/repos/greboid/ig/commits{/sha}",
    "git_commits_url": "https://api.github.com/repos/greboid/ig/git/commits{/sha}",
    "comments_url": "https://api.github.com/repos/greboid/ig/comments{/number}",
    "issue_comment_url": "https://api.github.com/repos/greboid/ig/issues/comments{/number}",
    "contents_url": "https://api.github.com/repos/greboid/ig/contents/{+path}",
    "compare_url": "https://api.github.com/repos/greboid/ig/compare/{base}...{head}",
    "merges_url": "https://api.github.com/repos/greboid/ig/merges",
    "archive_url": "https://api.github.com/repos/greboid/ig/{archive_format}{/ref}",
    "downloads_url": "https://api.github.com/repos/greboid/ig/downloads",
    "issues_url": "https://api.github.com/repos/greboid/ig/issues{/number}",
    "pulls_url": "https://api.github.com/repos/greboid/ig/pulls{/number}",
    "milestones_url": "https://api.github.com/repos/greboid/ig/milestones{/number}",
    "notifications_url": "https://api.github.com/repos/greboid/ig/notifications{?since,all,participating}",
    "labels_url": "https://api.github.com/repos/greboid/ig/labels{/name}",
    "releases_url": "https://api.github.com/repos/greboid/ig/releases{/id}",
    "deployments_url": "https://api.github.com/repos/greboid/ig/deployments",
    "created_at": 1526860662,
    "updated_at": "2020-03-23T11:24:36Z",
    "pushed_at": 1585081431,
    "git_url": "git://github.com/greboid/ig.git",
    "ssh_url": "git@github.com:greboid/ig.git",
    "clone_url": "https://github.com/greboid/ig.git",
    "svn_url": "https://github.com/greboid/ig",
    "homepage": "",
    "size": 2190,
    "stargazers_count": 0,
    "watchers_count": 0,
    "language": "Kotlin",
    "has_issues": true,
    "has_projects": true,
    "has_downloads": true,
    "has_wiki": true,
    "has_pages": false,
    "forks_count": 2,
    "mirror_url": null,
    "archived": false,
    "disabled": false,
    "open_issues_count": 14,
    "license": {
      "key": "mit",
      "name": "MIT License",
      "spdx_id": "MIT",
      "url": "https://api.github.com/licenses/mit",
      "node_id": "MDc6TGljZW5zZTEz"
    },
    "forks": 2,
    "open_issues": 14,
    "watchers": 0,
    "default_branch": "master",
    "stargazers": 0,
    "master_branch": "master"
  },
  "pusher": {
    "name": "dependabot-preview[bot]",
    "email": null
  },
  "sender": {
    "login": "dependabot-preview[bot]",
    "id": 27856297,
    "node_id": "MDM6Qm90Mjc4NTYyOTc=",
    "avatar_url": "https://avatars3.githubusercontent.com/in/2141?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/dependabot-preview%5Bbot%5D",
    "html_url": "https://github.com/apps/dependabot-preview",
    "followers_url": "https://api.github.com/users/dependabot-preview%5Bbot%5D/followers",
    "following_url": "https://api.github.com/users/dependabot-preview%5Bbot%5D/following{/other_user}",
    "gists_url": "https://api.github.com/users/dependabot-preview%5Bbot%5D/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/dependabot-preview%5Bbot%5D/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/dependabot-preview%5Bbot%5D/subscriptions",
    "organizations_url": "https://api.github.com/users/dependabot-preview%5Bbot%5D/orgs",
    "repos_url": "https://api.github.com/users/dependabot-preview%5Bbot%5D/repos",
    "events_url": "https://api.github.com/users/dependabot-preview%5Bbot%5D/events{/privacy}",
    "received_events_url": "https://api.github.com/users/dependabot-preview%5Bbot%5D/received_events",
    "type": "Bot",
    "site_admin": false
  },
  "created": false,
  "deleted": true,
  "forced": false,
  "base_ref": null,
  "compare": "https://github.com/greboid/ig/compare/4a22fe3bf1ff...000000000000",
  "commits": [

  ],
  "head_commit": null
}