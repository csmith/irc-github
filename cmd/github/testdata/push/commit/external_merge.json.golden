[greboid/ig] dependabot-preview[bot] pushed 0 commits to refs/heads/dependabot/gradle/org.jetbrains.kotlin-kotlin-gradle-plugin-1.3.71 - https://github.com/greboid/ig/compare/4a22fe3bf1ff...000000000000